src/foam/dao/FUIDDAO.js

```diff
@@ -128,6 +136,14 @@ foam.CLASS({
         }
         return getDelegate().put_(x, obj);
       `
+    },
+    {
+      name: 'cmd_',
+      args: 'Context x, Object obj',
+      javaCode:
+      `
+        return getSalt();
+      `
```

Comment:
FUID keys are only on lowest level daos, we need to find keys for all daos that can be accessed through the menuDAO

Reply:
Yes it will show both DAOs and lets the user pick. What do you mean by fall through?

Reply:
right it should check for SALT_CMD and otherwise call SUPER()

Reply:
> Yes it will show both DAOs and lets the user pick.

Then should it also consider the predicate on the menu eg. userDAO only shows User objects and businessDAO only shows Business objects?

Reply:
It does a find before showing the option to open it in the menu so if the find returns undefined it will hide the menu. L:77 of GlobalFUIDSearch.js

Reply:
ok, I see. Please add the check for SALT_CMD and otherwise call SUPER(). and can we have a demo of this tomorrow?